### PR TITLE
Add SCB to installments

### PR DESCRIPTION
**Pull request by alan-hadyk**
Test in `QuickStart.playground`:

```
func createCharge() {
    let currency = Currency.thb
    let params = ChargeParams(
        value: Value(
            amount: currency.convert(toSubunit: 6000.00),
            currency: currency
        ),
        sourceType: .installment(
            Installment.CreateParameter(
                brand: .scb,
                numberOfTerms: 6
            )
        ),
        isAutoCapture: false,
        returnURL: URL(string: "http://omise.co")
    )
    
    Charge.create(using: client, params: params) { (result) in
        switch result {
        case let .success(charge):
            print("created charge: \(charge.id) - \(charge.value.amount)")

        case let .failure(err):
            print("error: \(err)")
        }
    }
}
```

### CODE REVIEW

**Review by danfowler — APPROVED** [review]

**Comment by alan-hadyk** [comment]
Remark - tested successfully with other installment charges

**Review by igroomgrim — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -27,6 +27,9 @@
 .build/
 .fseventsd
 Carthage/Build
+Carthage/Checkouts
+Cartfile
```
Comment (nuxzero) [inline .gitignore]: @alan-hadyk Do we stop using Carthage? 
cc: @igroomgrim

**Review by nuxzero — COMMENTED** [review]
A small comment, rest of them look to me. 🙂

File: .gitignore (hunk 1 above)
Comment (igroomgrim) [reply .gitignore]: @nuxzero We still use it in some project and above just the rule for ignoring `Cartfile` and `Checkouts` folder.

Comment (nuxzero) [reply .gitignore]: I see. I just wondering why this PR ignore `Cartfile`. 🙂

**Review by nuxzero — APPROVED** [review]

**Review by tobias90 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (alan-hadyk) [reply .gitignore]: @nuxzero I had a lot of files when I've used Carthage - so I didn't want to commit them. Removed from `.gitignore`

**Pull request merged**